ui/apps/platform/src/Components/CompoundSearchFilter/types.ts

```diff
@@ -424,6 +478,9 @@ export const nodeComponentSearchFilterConfig = {
             filterChipLabel: 'Node Component Source',
             searchTerm: 'Component Source',
             inputType: 'select',
+            inputProps: {
+                options: ['GO', 'OS'],
```

Comment:
TODO: Need to double-check if this is correct. Went off mocks but they may be wrong. Will ask Charmik

Reply:
Likely all the values from https://github.com/stackrox/stackrox/blob/a8e7f23dbfdc668b74181631d040a29ef7a9ed52/proto/storage/image.proto#L144

We have this stored client side already too, but we should probably extract it out to the top level somewhere: https://github.com/stackrox/stackrox/blob/a8e7f23dbfdc668b74181631d040a29ef7a9ed52/ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/table.utils.ts#L44

Reply:
Sweet, thanks. I also asked Charmik for all the other ones, so I'll update the PR with the right options